src/components/Footer.js

```diff
@@ -178,10 +182,18 @@ function Footer({ classes }) {
             <a href="email.com" className={classes.email}>
               info@seasensors.org
             </a>
-            <FontAwesomeIcon icon={faFacebookF} color="#ffffff" size="2x" />
-            <FontAwesomeIcon icon={faTwitter} color="#ffffff" size="2x" />
-            <FontAwesomeIcon icon={faYoutube} color="#ffffff" size="2x" />
-            <FontAwesomeIcon icon={faGithub} color="#ffffff" size="2x" />
+            <a href="https://www.facebook.com/CodeForAfrica/">
+              <FontAwesomeIcon icon={faTwitter} color="#ffffff" size="2x" />
+            </a>
+            <a href="https://twitter.com/Code4Africa">
+              <FontAwesomeIcon icon={faFacebookF} color="#ffffff" size="2x" />
+            </a>
+            <a href="https://www.youtube.com/channel/UCa3qX2AwR9WDWO-7_MBL3Hg">
+              <FontAwesomeIcon icon={faYoutube} color="#ffffff" size="2x" />
+            </a>
+            <a href=" https://github.com/CodeForAfrica">
+              <FontAwesomeIcon icon={faGithub} color="#ffffff" size="2x" />
+            </a>
```

Comment:
All links should open on a blank tab with [`rel="noopener noreferrer"`](https://developers.google.com/web/tools/lighthouse/audits/noopener) and for anchors with text, lets use [Link](https://material-ui.com/style/links/) which adheres to Material UI theme settings.